sdk/keyvault/azure-keyvault/setup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from setuptools import setup
+
+with open("README.md", encoding="utf-8") as f:
+    README = f.read()
+with open("HISTORY.md", encoding="utf-8") as f:
+    HISTORY = f.read()
+
+setup(
+    name='azure-keyvault',
+    version='4.0.0b5',
+    description='Microsoft Azure Resource Management Client Libraries for Python',
```

Comment:
`description` should be updated to indicate that these are the KeyVault client libraries...

Reply:
fixed